pkg/hubble/metrics/metrics.go

```diff
@@ -53,6 +53,20 @@ var (
 	}, []string{labelSource})
 )
 
+// Metrics related to Hubble metrics HTTP requests handling
+var (
+	RequestsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: api.DefaultPrometheusNamespace,
+		Name:      "http_handler_requests_total",
```

Comment:
I think these names will be confused with the HTTP L7 metrics. Perhaps they should be prefixed with `metrics_`?

Reply:
Sure, prefixed names of the metrics with `metrics_`.

Changed names instead of the namespace for consistency, so that metrics declared by this file have the same metrics namespace (cannot change `lost_events_total` namespace due to backwards compatibility).